### PR TITLE
[Fix] update vnode_cache_size regularly to keep the metric correct.

### PR DESCRIPTION
**Pull request by liikeuforever**
# Required checklist
- [ ] Sample config files updated (`config`,`meta/config` and `default config`) 
- [ ] If there are user-facing changes, the documentation needs to be updated prior to approving the PR( [Link]() )
- [ ] If there are any breaking changes to public APIs, please add the `api change` label.
- [ ] Signed [CLA](https://cla-assistant.io/cnosdb/cnosdb) (if not already signed)

# Which issue does this PR close?
https://github.com/cnosdb/cnosdb/issues/1891
Currently metrics are stored once in 10s, so to ensure that the metric stays as valid as possible, it should be kept updated at a similar frequency.
[//]: # (We generally require a GitHub issue to be filed for all bug fixes and enhancements and this helps us generate change logs for our releases. You can link an issue to this PR using the GitHub syntax. For -- example `Closes #123` indicates that this PR will close issue #123.)

Related #.

# Rationale for this change

[//]: # (Why are you proposing this change? If this is already explained clearly in the issue then this section is not needed.
 Explaining clearly why changes are proposed helps reviewers understand your changes and offer better suggestions for fixes.)

# Are there any user-facing changes?

[//]: # (There is no need to duplicate the description in the issue here, but it is sometimes worth providing a summary of the individual changes in this PR.)

### CODE REVIEW

**Comment by Subsegment** [comment]
check failed

**Review by Subsegment — DISMISSED** [review]

File: tskv/src/tseries_family.rs [hunk 1]
```diff
@@ -1209,6 +1217,25 @@ impl TseriesFamily {
...
+                tsfamily.cache_size()
+            };
+            tsfamily
+                .write()
+                .await
+                .tsf_metrics
+                .record_cache_size(cache_size);
+        }
+    }
+
```
Comment (roseboy-liu) [inline tskv/src/tseries_family.rs:1238]: deal with tsfamily drop

Comment (liikeuforever) [reply tskv/src/tseries_family.rs:1246]: Use the weak pointer upgrade to check if the Arc reference is 0 and if the tsfamily is drop.

Comment (roseboy-liu) [reply tskv/src/tseries_family.rs:1246]: add test case for your code

File: tskv/src/mem_cache/series_data.rs [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+            return;
+        }
+
+        for item in self.groups.iter_mut() {
+            item.rows
+                .retain(|row| row.ts < range.min_ts || row.ts > range.max_ts);
+        }
+    }
+
+    pub fn read_data_v2(
```
Comment (Subsegment) [inline tskv/src/mem_cache/series_data.rs:117]: rename，read data

File: common/protos/src/generated/flatbuffers_generated/models.rs [hunk 3]
```diff
@@ -239,8 +239,8 @@ impl<'a> PingBody<'a> {
     PingBody { _tab: table }
   }
   #[allow(unused_mut)]
-  pub fn create<'bldr: 'args, 'args: 'mut_bldr, 'mut_bldr, A: flatbuffers::Allocator + 'bldr>(
```
Comment (Subsegment) [inline common/protos/src/generated/flatbuffers_generated/models.rs:242]: update flatbuffer version

**Comment by Subsegment** [comment]
add test case to slt for metric

**Review by Subsegment — APPROVED** [review]

**Pull request merged**